packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -921,6 +943,43 @@ export class DetailsListBase extends BaseComponent<IDetailsListProps, IDetailsLi
       onScroll(e);
     }
   };
+
+  private _getDetailsFooterProps(): IDetailsFooterProps | undefined {
+    const { adjustedColumns: columns } = this.state;
+    const selection = new Selection();
+    const {
+      compact,
+      selectionMode,
+      viewport,
+      checkboxVisibility,
+      getRowAriaLabel,
+      getRowAriaDescribedBy,
+      checkButtonAriaLabel,
+      checkboxCellClassName
+    } = this.props;
+    const groupNestingDepth = this._getGroupNestingDepth();
+    const footerRowProps: IDetailsRowProps | undefined = {
+      item: '',
+      itemIndex: -1,
+      compact: compact,
+      columns: columns as IColumn[],
+      selectionMode: selectionMode!,
+      selection: selection,
+      checkboxVisibility: checkboxVisibility,
+      getRowAriaLabel: getRowAriaLabel,
+      viewport: viewport,
+      getRowAriaDescribedBy: getRowAriaDescribedBy,
+      checkButtonAriaLabel: checkButtonAriaLabel,
+      checkboxCellClassName: checkboxCellClassName,
+      groupNestingDepth: groupNestingDepth
```

Comment:
>groupNestingDepth: groupNestingDepth [](start = 6, length = 36)

Is this required for list footer?

Reply:
yes, it is required for the footer to be aligned with the column headers/ row items